test/base/LoansBaseIsLoanSecuredTest.js

```diff
@@ -12,6 +12,9 @@ const Mock = artifacts.require("./mock/util/Mock.sol");
 const Loans = artifacts.require("./mock/base/LoansBaseMock.sol");
 const Settings = artifacts.require("./base/Settings.sol");
```

Comment:
Why the file test/base/LoansBaseGetCollateralInfoTest.js is empty?

Reply:
Should not be any more. We (@sypeer) were fixing the test